packages/block-library/src/comment-author-avatar/index.php

```diff
@@ -45,7 +61,10 @@ function render_block_core_comment_author_avatar( $attributes, $content, $block
 			'class'      => $classes,
 		)
 	);
-	return $avatar_string;
+	if ( isset( $spacing_attributes ) ) {
+		return sprintf( '<div style="%1s">%2s</div>', $spacing_string, $avatar_block );
```

Comment:
One thing that came up to my mind. Should we sanitize the computed string for the style?

Reply:
Yep, seems a good idea. Should I create a new PR, right?

Reply:
Yes, that's the best way to move forward.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/36988, thank you!